tensorflow-training/src/main/java/org/tensorflow/training/examples/MNISTTest.java

```diff
@@ -0,0 +1,331 @@
+/*
+ * Copyright (c) 2019, Oracle and/or its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.tensorflow.training.examples;
```

Comment:
I think it might be nice to have both the Logistic Regression MNIST you have, along with this more complex CNN example. Though we could just put them both in the same file, all the data loading, evaluation and training code will be the same.